feature/auth/src/main/java/com/example/auth/graph/signup/SignUpScreen.kt

```
@@ -0,0 +1,18 @@
+import androidx.compose.runtime.Composable
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.example.auth.graph.login.LoginViewModel
+
+@Composable
+internal fun SignUpRoute(
+    navigateToHome : () -> Unit,
+    viewModel: SignUpViewModel = hiltViewModel(),
+) {
+
+}
+
```

detekt (1.23.7)

[warning] The class or object SignUpViewModel is empty.

(detekt.empty-blocks.EmptyClassBlock)